nixos/modules/config/networking.nix

```diff
@@ -8,6 +8,12 @@ let
 
   cfg = config.networking;
 
+  upstreamNetworkUnits = [
+    "network.target"
+    "network-pre.target"
+    "network-online.target"
```

Comment:
Would the absent of these targets break many services? Since there are just text files do we really gain a lot by leaving them out?

Reply:
In order: The absence of these targets should break exactly those services which this new option is meant to disable; and leaving them out doesn't have anything to do with closure size, since they're shipped in systemd's `examples/` directory regardless. Does that help?